pom.xml

```diff
@@ -66,7 +66,6 @@
         <dependency.locations.enabled>false</dependency.locations.enabled>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <maven.compiler.release>8</maven.compiler.release>
```

Comment:
Required because as per the documentation for maven-compiler-plugin, the release flag is supported since 1.9, and thus has to be commented/removed from the POM if using earlier (1.8 and below) versions of JDK.

Maven documentation for release tag for maven-compiler-plugin can be found [here](https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html#release).